src/main/resources/templates/sightings.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html layout:decorate="layout"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <title>Sightings</title>
+    <link th:href="@{/css/sightings.css}" rel="stylesheet" />
+</head>
+<body class="page-container">
+<section class="sightings-section" layout:fragment="content">
+    <div class="sightings-heading">
+        <h1>Sightings</h1>
+    </div>
+    <div class="sightings-results-section">
+        <ul class="sightings-list">
+            <li class="results-list">
+                <div class="sightings-list" th:each="sighting: ${approvedSightings}">
```

Comment:
I don't think you have the `th:each` in the right place. I'd expect to see a document like the following:
```
<ul>
  <li>...</li>
  <li>...</li>
  ...
  <li>...</li>
<ul>
```
rather than like you have e.g. below:
```
<ul>
  <li>
     <div>...</div>
     <div>...</div>
      ....
     <div>...</div>
  </li>
</ul>
````